src/WireMock.Net.Abstractions/WireMock.Net.Abstractions.csproj

```diff
@@ -25,6 +25,14 @@
         <PackageLicenseExpression>MIT</PackageLicenseExpression>
     </PropertyGroup>
 
+    <PropertyGroup Condition="'$(TargetFramework)' == 'netstandard1.3' or '$(TargetFramework)' == 'netstandard2.0' or '$(TargetFramework)' == 'netstandard2.1'">
```

Comment:
Defining this constant is not needed. It is a default constant which is already available.